source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRButton.mm

```diff
@@ -9,22 +9,103 @@
 #import "ACOBaseActionElementPrivate.h"
 #import "ACOBundle.h"
 #import "ACOHostConfigPrivate.h"
+#import "ACRSVGImageView.h"
 #import "ACRUIImageView.h"
 #import "ACRViewPrivate.h"
-#import "ACRSVGImageView.h"
 #import "UtiliOS.h"
 
 @implementation ACRButton
 
+- (instancetype)initWithExpandable:(BOOL)expandable
+{
+    self = [super init];
+    if (self) {
+        if (expandable) {
+            [self setupExpandableConfig];
+        } else {
+            [self setupDefaultConfig];
+        }
+    }
+    return self;
+}
+
+- (void)setupExpandableConfig
+{
+    self.backgroundColor = UIColor.systemBlueColor;
+    self.tintColor = UIColor.systemBackgroundColor;
+
+    // Configure autoresizing mask
+    self.autoresizingMask = UIViewAutoresizingFlexibleWidth |
+                            UIViewAutoresizingFlexibleHeight;
+
+    // Set content insets (10 on all sides)
+    self.contentEdgeInsets = UIEdgeInsetsMake(10, 10, 10, 10);
+
+    // Set the title font (system font with a point size of 15)
+    self.titleLabel.font = [UIFont systemFontOfSize:15];
+
+    // Set images for various button states using SF Symbols (iOS 13+)
+    if (@available(iOS 13.0, *)) {
+        UIImage *chevronUp = [UIImage systemImageNamed:@"chevron.up"];
+        UIImage *chevronDown = [UIImage systemImageNamed:@"chevron.down"];
+        
+        [self setImage:chevronUp forState:UIControlStateNormal];
+        [self setImage:chevronUp forState:UIControlStateDisabled];
+        [self setImage:chevronDown forState:UIControlStateSelected];
+        [self setImage:chevronUp forState:UIControlStateHighlighted];
+    }
+
+    // Set title color for normal state to white
+    [self setTitleColor:[UIColor colorWithWhite:1 alpha:1] forState:UIControlStateNormal];
+
+    // Set corner radius
+    self.layer.cornerRadius = 10;
+
+    // Custom runtime attributes translated as properties
+    self.positiveForegroundColor = [UIColor colorWithWhite:0.6666666667 alpha:1.0];
```

Comment:
@GabrielMedAlv  @ardlank  
the colors used in this PR are the same as defined by xib before, it **should be fine** to let the pr merged with as is for now,
but the magic number like 0.6666666667 is not readable and maintainable, it should be a predefined value or even a theme observed value, at least using systems color pallete value that will change according to system themes